README.md

```diff
@@ -20,14 +20,18 @@ and relatively simple abstractions between HTTP API and storage backends.
 ```
 docker compose up -d
 ```
-2. Take note of the postgres connection string in the output of `docker compose
+2. Initialized the cockroachdb cluster:
+```
+docker exec -it portfolio-region1-1 ./cockroach init --insecure
+```
+3. Take note of the postgres connection string in the output of `docker compose
    logs`. These should look something like the following (note that you will
    have to replace the network identifier with `localhost` later):
 ```
 postgresql://root@71990c668326:26257/defaultdb?sslmode=disable
```

Comment:
For me just having `postgresql://root@localhost:26257/defaultdb?sslmode=disable` works